integration-libs/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -37,6 +37,12 @@ <h4>{{ item.name }}</h4>
       <div class="price">
         {{ item.price?.formattedValue }}
       </div>
+      <div class="price" *ngIf="item.stock?.stockLevel>0">
+        {{ item.stock?.stockLevelStatus }} : {{ item.stock?.stockLevel }}
+      </div>
+      <div class="price" *ngIf="item.stock?.stockLevel===undefined || item.stock?.stockLevel<=0">
```

Comment:
👍 

We can keep like this, but what do you think of wrapping it in an `else` clause instead.

For example,

```
      <div class="price" *ngIf="item.stock?.stockLevel > 0; else noStocks">
        {{ item.stock?.stockLevelStatus }} : {{ item.stock?.stockLevel }}
      </div>
      <ng-template #noStocks>
        <div class="price">
          {{ item.stock?.stockLevelStatus }}
        </div>
      </ng-template>
```

Reply:
you should run `yarn prettifer:fix` to fix the formatter of this template as it will throw an error in the CI